assignment-section-05/tests/test_prediction.py

```diff
@@ -21,4 +22,7 @@ def test_make_prediction(sample_input_data, first_twenty_expected_predictions):
     assert result.get("errors") is None
     assert len(predictions) == expected_no_predictions
     _predictions = list(predictions)
-    assert _predictions[:20] == first_twenty_expected_predictions
+    y_true = sample_input_data['survived']
+    accuracy = accuracy_score(_predictions, y_true)
+    assert accuracy > 0.7
```

Comment:
@solegalli presumably we want to aim for higher than this?

Reply:
If you were making a real model yes. But in the notebook we also have 0.71 and 0.72. So I would not bother too much. The idea was to introduce a variety of engineering steps and I think I used a logistic regression because I wanted to use a scaler to make the pipe a bit more complicated.  Trees do not need scaling of variables.